first-analysis-steps/davinci.md

```diff
@@ -118,22 +116,24 @@ above) and a set for everything else. Generally, you will want the latest
 version in the latter set, such as when making ntuples from Run 1 or Run 2
 data.
 
-These lessons use [DaVinci v44r6][v44r6], which was the latest Run 1/2 version at the
+These lessons use [DaVinci v45r1][v45r1], which was the latest Run 1/2 version at the
 time the text was last revised.
 
 [releases]: http://lhcbdoc.web.cern.ch/lhcbdoc/davinci/releases/
-[v44r6]: http://lhcbdoc.web.cern.ch/lhcbdoc/davinci/releases/v44r6/
+[v45r1]: http://lhcbdoc.web.cern.ch/lhcbdoc/davinci/releases/v45r1/
+
+**Note:** Older versions of DaVinci may not be available on the default platform ```x86_64-centos7-gcc8-opt```. To get around this we can pick the best suitable platform by using ```lb-run -c best DaVinci/vXXrYpZ ...```.
```

Comment:
Should we somewhere define what do we call a "platform", and what the name of the platform means?

Reply:
That's a good idea maybe with a link to [`HSF-TN-2018-01`](https://hepsoftwarefoundation.org/notes/HSF-TN-2018-01.pdf).